worlds/alttp/Rom.py

```diff
@@ -1440,14 +1444,14 @@ def chunk(l, n):
     rom.write_bytes(0x183000, equip[0x340:])
     rom.write_bytes(0x271A6, equip[0x340:0x340 + 60])
 
-    rom.write_byte(0x18004A, 0x00 if world.mode[player] != 'inverted' else 0x01)  # Inverted mode
+    rom.write_byte(0x18004A, 0x00 if world.world_state[player] != WorldState.option_inverted else 0x01)  # Inverted mode
     rom.write_byte(0x18005D, 0x00)  # Hammer always breaks barrier
-    rom.write_byte(0x2AF79, 0xD0 if world.mode[
-                                        player] != 'inverted' else 0xF0)  # vortexes: Normal  (D0=light to dark, F0=dark to light, 42 = both)
-    rom.write_byte(0x3A943, 0xD0 if world.mode[
-                                        player] != 'inverted' else 0xF0)  # Mirror: Normal  (D0=Dark to Light, F0=light to dark, 42 = both)
-    rom.write_byte(0x3A96D, 0xF0 if world.mode[
-                                        player] != 'inverted' else 0xD0)  # Residual Portal: Normal  (F0= Light Side, D0=Dark Side, 42 = both (Darth Vader))
+    rom.write_byte(0x2AF79, 0xD0 if world.world_state[
+                                        player] != WorldState.option_inverted else 0xF0)  # vortexes: Normal  (D0=light to dark, F0=dark to light, 42 = both)
```

Comment:
should probably have a local variable `inverted = ...` to make this more readable
that's used a ton of times above and below as well.

Reply:
writing a property when you're almost done needing it and then not going back to use it? couldn't be me.